logback-config-test/src/main/java/cn/home1/logging/logback/LogbackApplication.java

```diff
@@ -0,0 +1,26 @@
+package cn.home1.logging.logback;
```

Comment:
Module logback-config-test is not the tests jar of logback-config-logstash.
It provides a test scope logback config file (logback-test.xml or logback-spring-test.xml) for moduls that run tests.
You should write tests in logback-config-test directly or put them in a new module.